src/org/org.ts

```diff
@@ -755,6 +764,90 @@ export class Org extends AsyncOptionalCreatable<Org.Options> {
     );
   }
 
+  /**
+   * Query SandboxInfo Features by sandbox name
+   *
+   * @param sandboxName The sandbox name to query for
+   */
+  public async querySandboxInfoFeaturesBySandboxName(name: string): Promise<string | undefined> {
+    return (
+      await this.connection.singleRecordQuery<{ Features: string }>(
+        `SELECT Features FROM SandboxInfo WHERE SandboxName = '${name}'`,
+        { tooling: true }
+      )
+    ).Features;
+  }
+
+  /**
+   * Query SandboxInfo Features by ID
+   *
+   * @param id The SandboxInfo ID to query for
+   */
+  public async querySandboxInfoFeaturesById(id: string): Promise<string | undefined> {
```

Comment:
This method is not needed and should be deleted.  `querySandboxInfo()` should be used instead.